src/resolver/RegisterInternship.ts

```diff
@@ -0,0 +1,23 @@
+import {Db} from "mongodb";
+import {InternshipOffer} from "./InternshipOffer";
+
+export const registerInternshipResolver = (db: Db) => async ({input}) => {
+    const doc = {
+        description: input.description,
+        jobname: input.jobname,
+        location: {
+            street: input.addressStreet,
+            plz: input.addressPLZ,
+            town: input.addressTown
+        },
+        contact: {
+            name: input.contactName,
+            email: input.contactEmail,
+            phone: input.contactPhone
+        },
+        link: input.link
+    };
+    let result = await db.collection('Internships').insertOne(doc);
```

Comment:
Es fehlt noch die Autorisierung, ob der aktuelle User überhaupt angemeldet ist und Angebote posten darf.

Das erfordert aber eine andere Version von Graphiql (das aktuelle Frontend), das es unterstützt Tokens mitzugeben, ist also soweit mal in Ordnung, muss aber später noch bedacht werden.